packages/lit-element/src/lib/updating-element.ts

```diff
@@ -577,14 +576,14 @@ export abstract class UpdatingElement extends HTMLElement {
     const ctor = this.constructor as typeof UpdatingElement;
     // Note, hint this as an `AttributeMap` so closure clearly understands
     // the type; it has issues with tracking types through statics
-    // tslint:disable-next-line:no-unnecessary-type-assertion
```

Comment:
I think we could keep the `no-unnecessary-type-assertion` rule, and remove the cast here.

Reply:
It is not necessary with ESLint.